kosat-core/src/commonMain/kotlin/org/kosat/SolverTypes.kt

```diff
@@ -81,6 +85,10 @@ value class Var(val index: Int) {
 
     /** A literal of negation of that variable */
     val negLit: Lit get() = Lit((index shl 1) or 1)
+
+    override fun toString(): String {
```

Comment:
Hmmm, `Var`iables inside solver are always 0-based, and here you are distracting users/devs (note that this is not a contrived problem!) by showing them 1-based index. If you want 1-based variable, convert it via `toDimacs` and format however you like. Here, `Var` is 0-based, and its `toString` should be either `inner.toString()` or `"Var($inner)"`.